components/central-application-gateway/README.md

```diff
@@ -91,18 +95,18 @@ For instance, if the user registered the `cc-occ` API bundle with the `commerce-
 http://central-application-gateway.kyma-system:8082/ec/cc-occ/commerce-webservices/basesites
 ```
 
-As a result, the Central Application Gateway:
-1. Looks for the `cc-occ` service and the `commerce-webservices` entry in the `ec` Application CRD and extracts the target URL path along with the authentication configuration
-2. Modifies the request to include the authentication data
+As a result, Central Application Gateway:
+1. Looks for the `cc-occ` service and the `commerce-webservices` entry in the `ec` Application CR and extracts the target URL path along with the authentication configuration.
+2. Modifies the request to include the authentication data.
 3. Sends the request to the following path: 
    ```bash
    {TARGET_URL_EXTRACTED_FROM_APPLICATION_CRD}/basesites
    ```
 
 #### Handling ambiguous API definition names
 
-A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CRD should be unique for a given application.
-Invocation of endpoints with duplicate names will result in a **400 Bad Request** failure. In such a case, one of the names should be changed to avoid ambiguity.
+A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CR should be unique for a given application.
+Invocation of endpoints with duplicate names will result in a `400 Bad Request` failure. In such a case, one of the names must be changed to avoid ambiguity.
```

Comment:
```suggestion
Invocation of endpoints with duplicate names results in the `400 Bad Request` failure. In such a case, change one of the names to avoid ambiguity.
```

Reply:
Again, I'll apply just the tense part. I'll also leave `must` because it puts emphasis on the required action.